certified-connectors/BoldSign/README.md

```diff
@@ -0,0 +1,31 @@
+# Introduction
+
```

Comment:
Please update the readme.md file as per the template below. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

Reply:
Updated readme as per template